resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/repository/JpaPersistenceLayer.java

```diff
@@ -398,107 +416,106 @@ private void saveSerialisedFormat(final String serialisedFormat, final LeafResou
      * Get a single resource by resource id with all parents resolved - not necessarily a leaf
      *
      * @param resourceId the id of the resource to get
-     * @return Optional.of a {@link Resource} in persistence, Optional.empty if not found
+     * @return {@link Iterator} of a {@link Resource} in persistence, Optional.empty if not found
```

Comment:
method is no longer dealing with Optionals, update javadoc

Reply:
Updated the JavaDoc to not include `Optional` reference